game/scripts/vscripts/items/bloodmage.lua

```diff
@@ -0,0 +1,143 @@
+---
+--- Created by Zarnotox.
+--- DateTime: 03-Dec-17 21:32
+---
+
+item_bloodmage = class(ItemBaseClass)
+
+LinkLuaModifier( "modifier_item_bloodmage_sangromancy", "items/bloodmage.lua", LUA_MODIFIER_MOTION_NONE )
+
+--------------------------------------------------------------------------------
+
+function item_bloodmage:GetAbilityTextureName()
+  local baseName = self.BaseClass.GetAbilityTextureName( self )
+
+  local activeName = ""
+
+  if self.mod and not self.mod:IsNull() then
+    activeName = "_active"
+  end
+
+  return baseName .. activeName
+end
+
+--------------------------------------------------------------------------------
+
+function item_bloodmage:GetIntrinsicModifierName()
+  return "item_bloodmage"
```

Comment:
This should return the name of a modifier that grants the item's passive stats. Typically `modifier_{itemName}`, so `modifier_item_bloodmage` here.
Note that passive item modifiers should have `IsPurgable` false, usually `IsHidden` true, and a `GetAttributes` method that returns `MODIFIER_ATTRIBUTE_MULTIPLE`, so that there can be multiple instances of the modifier on the same unit. (I wonder if we shouldn't make a base class for those really)

Reply:
Ah I understand, I'll be sure to change that when adding actual stats